README.md

```diff
@@ -92,8 +92,8 @@ desktop = Sandbox()
 # Start the stream
 desktop.stream.start()
 
-# Get stream URL
-url = desktop.stream.get_url()
+# Get stream URL and disable user interaction
+url = desktop.stream.get_url(view_only=True)
```

Comment:
instead of replacing the current example, copy the new one below the existing one